projects/packages/classic-theme-helper/src/class-featured-content.php

```diff
@@ -753,7 +758,7 @@ function wpcom_rest_api_featured_content_copy_plugin_actions( $copy_dirs ) {
 		$copy_dirs[] = __FILE__;
 		return $copy_dirs;
 	}
-	add_action( 'restapi_theme_action_copy_dirs', 'wpcom_rest_api_featured_content_copy_plugin_actions' );
+	add_action( 'restapi_theme_action_copy_dirs', __NAMESPACE__ . '\wpcom_rest_api_featured_content_copy_plugin_actions' );
```

Comment:
Should we move these two hooks which are related to WPCOm somewhere else?

Also `restapi_theme_action_copy_dirs` seems to be a filter, not an action. Is it possible that this never worked properly?

Reply:
If they're only useful on WordPress.com Simple, I think we could move them to the mu-plugin loading the feature on simple.
Related discussion: p1715084696384109-slack-C05PV073SG3